### DISCUSSION TITLE
About syntax coloring

### DISCUSSION THREAD

**Discussion by Mishasama** [SubtitleEdit/subtitleedit#7354] [Q&A]
![image](https://github.com/SubtitleEdit/subtitleedit/assets/54498172/6b42339a-a0e9-468e-a822-347e56ab6b68)
I don't know what the pixels mean. 
```
1920 ÷ 45 = 42 characters ≤ 43 characters = Red?
``` 
![image](https://github.com/SubtitleEdit/subtitleedit/assets/54498172/d9ae332c-2382-4368-812a-5e83070e9f03)
It looks like it says it has exceeded the screen.
![image](https://github.com/SubtitleEdit/subtitleedit/assets/54498172/bd0f75c4-556b-4f05-9f20-b78e9cb96adf)
But actually, it's not. This is an FHD video with 1920 width.

**Comment by Flitskikker** [comment]
@Mishasama 
What is the font name & font size here?

![](https://user-images.githubusercontent.com/54498172/266749073-bd0f75c4-556b-4f05-9f20-b78e9cb96adf.png)

**Reply by Mishasama** [reply]
Same as settings.
